packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3283,7 +3308,13 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
             !TableUtils.isBigDecimalType(selectedColumn.type) &&
             !TableUtils.isBigIntegerType(selectedColumn.type)
           ) {
-            const inputValue = parseInt(inputString, 10);
+            let inputValue = parseInt(inputString, 10);
+            if (inputString === '-Infinity') {
```

Comment:
In another spot you allow for `-i` and `-infinity` as valid input strings. Why not here? Or is whatever the user types in those cases replaced w/ `-Infinity`

Reply:
in the other spot, -i is automatically changed to -infinity (kind of like auto-completing). So I'm assuming the only possible sequence of letters in this input is either `Infinity` or `-Infinity` since I'm not allowing any other strings